src/phases/summon-phase.ts

```diff
@@ -195,6 +195,10 @@ export class SummonPhase extends PartyMemberPokemonPhase {
                 pokemon.cry(pokemon.getHpRatio() > 0.25 ? undefined : { rate: 0.85 });
                 pokemon.getSprite().clearTint();
                 pokemon.resetSummonData();
+                // necessary to stay transformed during wild waves
+                if (pokemon.summonData?.speciesForm) {
+                  pokemon.loadAssets();
```

Comment:
```suggestion
                  pokemon.loadAssets(false);
```
This should fix the sprite loading issues I believe.

Reply:
Yes, I'll make those changes